RubeesEat/Pages/SplitBill.cshtml

```diff
@@ -10,7 +10,7 @@ Split bill:
     <label for="billAmount">Total Price:</label>
     <input id="billAmount" type="number" step=".01" name="billAmount" required="required"><br>
 
-    <span id="addedPeople"></span>
+    <label id="addedPeople"></label>
```

Comment:
`<label>` should only be used together with another element that is referenced using the `for` attribute. Here we just want a container for elements so we should use `<div>` instead.